src/main/java/persistence/entity/impl/retrieve/EntityCollectionLoader.java

```diff
@@ -30,23 +36,41 @@ public <T> T loadCollection(Class<T> clazz, Object instance, ColumnType columnTy
         for (ColumnMeta columnMeta : relationColumnMetaList) {
             if (columnMeta.isEagerLoading()) {
                 eagerLoad(instance, columnMeta, objectMappingMeta, columnType);
+                continue;
             }
 
-            // TODO: lazy load proxy
+            lazyLoad(instance, columnMeta, objectMappingMeta, columnType);
         }
 
         return clazz.cast(instance);
     }
```

Comment:
요부분만 load 와 같은 private 메소드로 추출하면 조금더 깔끔하게 다듬을 수 있을것 같아요
```suggestion
        relationColumnMetaList.forEach(columnMeta -> load(instance, columnMeta, objectMappingMeta, columnType));
        return clazz.cast(instance);
    }
        
    private void load(Object instance, ColumnMeta columnMeta, EntityObjectMappingMeta objectMappingMeta, ColumnType columnType) {
        if (columnMeta.isEagerLoading()) {
            eagerLoad(instance, columnMeta, objectMappingMeta, columnType);
            return;
        }
        lazyLoad(instance, columnMeta, objectMappingMeta, columnType);
    }
```
요부분만 load 와 같은 private 메소드로 추출하면 조금더 깔끔하게 다듬을 수 있을것 같아요